src/components/calendar/ClassModal.js

```diff
@@ -26,6 +28,21 @@ function ClassModal({ showDialog, toggleDialog, classes, section, removeSection
         <Typography variant="h5">
           {`${section.subjectId} ${section.courseId} Section ${section.sectionNumber}`}
         </Typography>
+        <Typography>
+          {`${section.schedules.map(getFormattedClassSchedule)}`}
+        </Typography>
+        <Typography>
+          {`${section.schedules.map(schedule => schedule.location)}`}
+        </Typography>
+        <Typography gutterBottom="true">
+          {`${section.instructors.map(instructor => instructor)}`}
+        </Typography>
+        <Typography variant="h6">
+          {`${section.descriptions.map(description => description.name)}`}
```

Comment:
This won't work because if there is more than one description, you'd see all the description names, *and then* you'd see all the description texts. You want one after the other in pairs. See if you can figure out how to do that - I think addressing my comment above about returning JSX for each element of the map should help you.